2023-04-19 00:49:02,488 TADA INFO starting test `direct_ldms_ls_conn_test`
2023-04-19 00:49:02,488 TADA INFO   test-id: 5f993094fb00ba679ee888ae8c5e880f6063bb0fbc9306b0ee42af7bbc10cf56
2023-04-19 00:49:02,488 TADA INFO   test-suite: LDMSD
2023-04-19 00:49:02,488 TADA INFO   test-name: direct_ldms_ls_conn_test
2023-04-19 00:49:02,488 TADA INFO   test-user: narate
2023-04-19 00:49:02,489 TADA INFO   commit-id: 29518afaedb92e5267889f02978b2a5e42f5de6b
2023-04-19 00:49:02,724 __main__ INFO starting munged on cygnus-01-iw
2023-04-19 00:49:03,036 __main__ INFO starting munged on localhost
2023-04-19 00:49:03,269 __main__ INFO starting ldmsd on cygnus-01-iw
2023-04-19 00:49:03,608 TADA INFO assertion 0, Start ldmsd sampler and munged: OK, passed
2023-04-19 00:49:08,716 TADA INFO assertion 1, ldms_ls to the sampler: ldms_ls failed, output:
Wed Apr 19 00:49:08 2023:    ERROR: auth_munge: munge_decode() failed. Invalid credential
Connection failed/rejected.
, failed
Traceback (most recent call last):
  File "direct_ldms_ls_conn_test", line 281, in <module>
    raise RuntimeError("ldms_ls rc: {}".format(rc))
RuntimeError: ldms_ls rc: 1
2023-04-19 00:49:08,718 TADA INFO assertion 2, Kill the sampler: skipped
2023-04-19 00:49:08,718 TADA INFO assertion 3, ldms_ls to the dead sampler: skipped
2023-04-19 00:49:08,718 TADA INFO assertion 4, ldms_ls to a dead host: skipped
2023-04-19 00:49:08,719 TADA INFO test direct_ldms_ls_conn_test ended
2023-04-19 00:49:08,927 __main__ INFO stopping munged on cygnus-01-iw
2023-04-19 00:49:09,346 __main__ INFO stopping munged on localhost
